gitpod-io/gitpod
# Allow manually disabling prebuilds for a project

## Description


Allow manually disabling prebuilds for a project.

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/15281

## How to test


1. Create a project, but don't enable prebuilds
2. Push a new commit to the project's repository
3. There should be zero prebuilds in the project

## Release Notes

```release-note
Allow manually disabling prebuilds for a project
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
